src/components/DataCard/index.jsx

```diff
@@ -0,0 +1,50 @@
+import { Component } from 'react';
+import Card from '@material-ui/core/Card';
+import { withStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import { arrayOf, object } from 'prop-types';
+
+@withStyles(theme => ({
+  paper: {
+    minHeight: 100,
+    paddingBottom: theme.spacing.unit,
+  },
+  '@media screen and (max-width: 900px)': {
```

Comment:
Use material UI breakpoints instead of hardcoding values here: https://material-ui.com/layout/breakpoints/#breakpoints.

If you are unsatisfied with the default breakpoints, you can always configure them in `theme.jsx`.